content/en/about-filecoin/how-filecoin-works.md

```diff
@@ -81,11 +81,11 @@ Filecoin implementations may choose the heuristics of how their storage provider
 
 ## Actors
 
-Actors are a [software design pattern](https://en.wikipedia.org/wiki/Actor_model) for managing state. Accounts, Multisigs, Miners, and anything with a state, such as an account balance, are implemented as an _actor_.
+Actors are a [software design pattern](https://en.wikipedia.org/wiki/Actor_model) for managing state. Accounts, Multisigs, Storage Providers, and anything with a state, such as an account balance, are implemented as an _actor_.
 
 Actors are the Filecoin equivalent of smart contracts in the Ethereum Virtual Machine. As such, Actors are very central components of the system. Any change to the current state of the Filecoin blockchain has to be triggered through an Actor.
 
-see: [https://spec.filecoin.io/#section-systems.filecoin_vm](https://spec.filecoin.io/#section-systems.filecoin_vm)
+See: [Filecoin spec > Virtual Machine](https://spec.filecoin.io/#section-systems.filecoin_vm)
```

Comment:
Simply use the title of the page when linking to it. We'd only really use abc > efg > hij when telling a user where to click. And even then we'd use the right arrow symbol →.